src/main/java/TypewiseAlert/TriggerAlert.java

```diff
@@ -0,0 +1,21 @@
+package TypewiseAlert;
+
+public class TriggerAlert {
+	
+	 public static void sendToController(BreachTypeEnum breachType) {

```

Comment:
using static methods could lead to multithreading issues. Static needs to be carefully used.